app/parsers/powershell.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from app.utility.base_parser import BaseParser, PARSER_SIGNALS_FAILURE
+
+
+class Parser(BaseParser):
+
+    def parse(self, blob):
+        for ex_line in self.line(blob):
+            if 'FullyQualifiedErrorId' in ex_line:
```

Comment:
Maybe pull out static string into global list of strings to check for?

Reply:
Yeah, that makes sense to do.